interfaces/builtin/account_control_test.go

```diff
@@ -35,9 +36,23 @@ type AccountControlSuite struct {
 	plug  *interfaces.Plug
 }
 
-var _ = Suite(&AccountControlSuite{
-	iface: builtin.NewAccountControlInterface(),
-	slot: &interfaces.Slot{
+var _ = Suite(&AccountControlSuite{})
+
+const accountCtlMockPlugSnapInfo = `name: other
+version: 1.0
+plugs:
```

Comment:
This `plugs:` subtree is redundant AIUI, if the plug name is the name of an existing interface it does not need to be declared explicitly. Having said that, maybe we still want to keep it so that things are explicit.